2019_Spring/readme.md

```diff
@@ -38,7 +38,7 @@ I suggest you @mention me (@joewadcan) in the Pull Request, so we can chat and f
 
 - [X] Blas, Arion :rocket:
 
-- [ ] Briggs, Andrew
+- [X ] Briggs, Andrew 🚅 
```

Comment:
Hey @anbriggs looks like you added an extra space after that `x` - can you edit & remove it?